### PR TITLE
Update gir files

### PR DESCRIPTION
**Pull request by alatiera**
Glib girs are held back until upstream issue is resolved.

https://github.com/gtk-rs/gir-files/pull/114#discussion_r876844419

### CODE REVIEW

File: Pango-1.0.gir [hunk 1]
```diff
@@ -4358,6 +4358,14 @@ for enumerating faces.</doc>
           </parameter>
         </parameters>
       </method>
+      <property name="item-type" transfer-ownership="none">
```
Comment (bilelmoussaoui) [inline Pango-1.0.gir:4361]: needs since annotation upstream iirc

**Comment by sdroege** [comment]
Also needs gtk-rs-core regenerated for pango

**Comment by alatiera** [comment]
Upstream MRs:

https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4815

https://gitlab.gnome.org/GNOME/pango/-/merge_requests/624

**Comment by alatiera** [comment]
Since only the pango changes where left and they only affect gtk-rs-core, I split them into a separate MR we can merge once the upstream MR is reviewed and merged.

File: Gtk-4.0.gir [hunk 2]
```diff
@@ -73519,6 +74180,9 @@ when the widget has focus.</doc>
           </parameter>
         </parameters>
       </method>
+      <property name="item-type" transfer-ownership="none">
```
Comment (bilelmoussaoui) [inline Gtk-4.0.gir:74183]: Sorry I missed this one, but it lacks a since annotation upstream

Comment (alatiera) [reply Gtk-4.0.gir:74183]: Fixed upstream https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/4827

**Comment by bilelmoussaoui** [comment]
looks good to me, thanks!

**Pull request merged**